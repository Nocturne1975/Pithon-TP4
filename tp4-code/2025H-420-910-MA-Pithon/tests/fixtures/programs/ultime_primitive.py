nums = [1, 2, 3, 4, 5]
total = 0

for n in range(len(nums)):
    if nums[n] % 2 ==0:
        continue
    total = total + nums[n]

print("Somme des impairs:", total)

#Comparaisons

a = 10
b = 5
print(a > b)
print(a < b)
print(a == b)
print(a != b)
print(a >= b)
print(a <= b)

#Manipulations des chaines

s = "pithon"
print (upper(s))
print(str(len(s)) + " lettres")

# Liste des tuples

t = (1, 2)
l = [3, 4]
print(concat(t, tuple(l)))
print(append(nums, 6))
print(head(nums))
print(tail(nums))
