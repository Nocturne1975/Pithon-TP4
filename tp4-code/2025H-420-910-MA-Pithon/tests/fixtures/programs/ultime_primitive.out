Somme des impairs: 9
True
False
False
True
True
False
PITHON
6 lettres
(1, 2, 3, 4)
[1, 2, 3, 4, 5, 6]
1
[2, 3, 4, 5]
